tls/s2n_resume.c

```diff
@@ -684,7 +686,7 @@ int s2n_compute_weight_of_encrypt_decrypt_keys(struct s2n_config *config,
         }
     }
 
-    POSIX_BAIL(S2N_ERR_ENCRYPT_DECRYPT_KEY_SELECTION_FAILED);
+    POSIX_BAIL(S2N_ERR_NO_TICKET_ENCRYPT_DECRYPT_KEY);
```

Comment:
This seems unnecessary. What's wrong with the existing error?
https://github.com/aws/s2n-tls/blob/1ed5624c24c80b5ef93f93a72ee93b503ddd6eba/error/s2n_errno.c#L235-L236

Reply:
Oops this was not intentional. Will change it back